bbk-frontend/app/[recipeId]/page.tsx

```diff
@@ -1,20 +1,36 @@
 import { dummyRecipes } from "@/data/recipes";
 import Image from "next/image";
 
-export default function Page({ params }: { params: { recipeId: number } }) {
-  const recipeData = dummyRecipes.find(
-    (recipe) => recipe.id == params.recipeId,
-  );
+async function getRecipeDetails(recipeId: number) {
+  const getRecipesPath = process.env.API_PATH + "recipes/" + recipeId;
+
+  const res = await fetch(getRecipesPath);
+  if (!res.ok) {
+    //what solution for errors ? Message component or some kind of notification
```

Comment:
we can add the toast component from shadcn, otherwise, we can add an `error.tsx` page